src/liboslexec/shadingsys.cpp

```diff
@@ -2692,28 +2692,23 @@ ShadingSystemImpl::optimize_group (ShaderGroup &group,
 
 
 
-static void optimize_all_groups_wrapper (ShadingSystemImpl *ss, int mythread, int totalthreads)
-{
-    ss->optimize_all_groups (1, mythread, totalthreads);
-}
-
-
-
 void
 ShadingSystemImpl::optimize_all_groups (int nthreads, int mythread, int totalthreads)
 {
     // Spawn a bunch of threads to do this in parallel -- just call this
     // routine again (with threads=1) for each thread.
     if (nthreads < 1)  // threads <= 0 means use all hardware available
-        nthreads = std::min ((int)boost::thread::hardware_concurrency(),
+        nthreads = std::min ((int)OIIO::Sysutil::hardware_concurrency(),
```

Comment:
No, just a wrapper dating from before C++11. I guess I can skip the middleman here and to straight for std::thread::hardware_concurrency().